### ISSUE TITLE
GC logo gets listed as an WCAG SC 1.1.1 error

### ISSUE THREAD

**Issue by bsouster** [wet-boew/GCWeb#1670]
@duboisp @GormFrank 

This code is creating a false positive in an automatic accessibility testing tool being used by GC:

    <a href="/en.html" property="url">
        <img src="/etc/designs/canada/wet-boew/assets/sig-blk-en.svg" alt="" property="logo">
        <span id="" class="wb-inv" property="name"> Government of Canada</span>
    </a>

The technique was necessary at the time it was coded but can now be updated with techniques automatic testing tools recognize:

    <a href="/en.html" property="url">
        <img src="/etc/designs/canada/wet-boew/assets/sig-blk-en.svg" alt="Government of Canada" property="logo">
    </a>

There are suggestions that role="img" is necessary for some adaptive technologies but I've yet to find out which AT those are but I'll look around.

**Comment by bsouster** [comment]
There is discussion whether role="img" should be applied to images with SVG src. The origin of this as a recommendation comes from [Deque's research on accessible SVG patterns](https://codepen.io/cariefisher/pen/bOKEVQ).

The standard (img+src+alt) pattern with SVGs does not work with Mobile (iOS): Safari + VoiceOver
(v12.1.2).

@duboisp : you said you want more research on this before applying role fixes
